stylesheets/docs.css

```diff
@@ -0,0 +1,150 @@
+html {
+  scroll-behavior: smooth;
+}
+
+/* each component section */
+section {
+  border-bottom: 1px solid grey;
+  padding: 1rem;
+  padding-top: 2.3rem;
+}
+
+/* navigation bar */
+#navigation {
+  grid-area: navbar;
+  width: 100%;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+}
+
+/* sidebar list */
+#main-list {
+  position: fixed;
+  top: 7.6%;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  overflow-y: auto;
+  height: calc(100vh - 1rem);
+}
+
+/* sidebar list item */
+#main-list > .list-item {
+  padding: 0;
+  margin: 0;
+  text-align: center;
+}
+#main-list > .list-item > a {
```

Comment:
don't make high Specificity. please try to make your code with one class and two class only